Jellyfin.Api/Controllers/PluginsController.cs

```diff
@@ -64,16 +203,27 @@ public ActionResult<IEnumerable<PluginInfo>> GetPlugins()
         [Authorize(Policy = Policies.RequiresElevation)]
         [ProducesResponseType(StatusCodes.Status204NoContent)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [Obsolete("Please use the UninstallPluginByVersion API.")]
         public ActionResult UninstallPlugin([FromRoute, Required] Guid pluginId)
         {
-            var plugin = _appHost.Plugins.FirstOrDefault(p => p.Id == pluginId);
+            // If no version is given, return the current instance.
+            var plugins = _pluginManager.Plugins.Where(p => p.Id.Equals(pluginId));
+
+            // Select the un-instanced one first.
+            var plugin = plugins.FirstOrDefault(p => p.Instance != null);
```

Comment:
The comment doesn't seem to fit. If the check is `Instance != null` doesn't that make it an _instanced_ plugin?

Reply:
Fixed,